### ISSUE TITLE
RelevantFeatureAugmenter-evaluate_only_added_features=True does not function as expected

### ISSUE THREAD

**Issue by mgulcin** [blue-yonder/tsfresh#159]
I have some pre-extracted features and I want to use these features together with features extracted and selected by RelevantFeatureAugmenter.  In order to achieve this I set "evaluate_only_added_features = True" . However after fit and transform, the output features that I've obtained don't contain pre-extracted features.  

I tried to use FeatureAugmenter and it works fine. Am I missing something or is there a small bug in RelevantFeatureAugmenter?

Here is an example code:

    download_robot_execution_failures()
    df, y = load_robot_execution_failures()
    # leave only a single column, namely a
    df = df.drop(['b', 'c','d','e','f'],axis=1)
    
    # Here, df contains the time series of both train and test set. 
    # We will split it into a train df_train and a test set  df_test:
    y_train, y_test = train_test_split(y)
    df_train = df.loc[df.id.isin(y_train.index)]
    df_test = df.loc[df.id.isin(y_test.index)]
    X_train = pd.DataFrame(index=y_train.index)
    X_test = pd.DataFrame(index=y_test.index)
    df_train.shape, df_test.shape
    
    X_train["pre_extracted_feature"] = 0 # add a feature
    df_train["pre_extracted_feature"] = 0

    fresh = RelevantFeatureAugmenter(column_id='id', column_sort='time', evaluate_only_added_features = True)
    fresh.timeseries_container=df_train
    fresh.fit(X_train, y_train)
    # print - Note that "pre_extracted_feature" is not listed
    print list(fresh.feature_selector.relevant_features)

    transformed = fresh.transform(X_train)
    # print - Note that "pre_extracted_feature" is not listed
    print list(transformed.columns)

**Comment by MaxBenChrist** [comment]
Missing important statements in your snippet:

```
import pandas as pd
from tsfresh.examples.robot_execution_failures import load_robot_execution_failures, download_robot_execution_failures
from sklearn.model_selection import train_test_split
import numpy as np
from tsfresh.transformers.relevant_feature_augmenter import RelevantFeatureAugmenter
```

You should not add your features to the time series container `df`. So please delete the line 
```
df_train["pre_extracted_feature"] = 0
```
in your snippet

Otherwise I could reproduce your bug. I will look into that now

**Comment by MaxBenChrist** [comment]
See the pr https://github.com/blue-yonder/tsfresh/pull/160

It should get in the master after review. Then we will have a new version on pypi.

**Comment by mgulcin** [comment]
Thank you.
